ABRSchema_0-2-0.json

```diff
@@ -634,6 +643,61 @@
                     }
                 },
                 "additionalProperties": false
+            },
+            "Volumes": {
+                "$comment": "Update Plates whenever you add a new plate type. Inputs should have a `inputType` and `parameterName`.",
+                "type": "object",
+                "properties": {
+                    "Key Data": {
+                        "properties": {
+                                "inputType": { "const": "IVLab.ABREngine.VolumeKeyData" },
+                                "parameterName": { "const": "Key Data" },
+                                "inputGenre": { "const": "KeyData" },
+                                "inputValue": { "$ref": "#/definitions/InputStringTypes/DataPath" }
+                        }
+                    },
+                    "Color Variable": {
+                        "properties": {
+                                "inputType": { "const": "IVLab.ABREngine.ScalarDataVariable" },
+                                "parameterName": { "const": "Color" },
+                                "inputGenre": { "const": "Variable" },
+                                "inputValue": { "$ref": "#/definitions/InputStringTypes/DataPath" }
+                        }
+                    },
+                    "Colormap": {
+                        "properties": {
+                                "inputType": { "const": "IVLab.ABREngine.ColormapVisAsset" },
+                                "parameterName": { "const": "Color" },
+                                "inputGenre": { "const": "VisAsset" },
+                                "inputValue": { "$ref": "#/definitions/InputStringTypes/UUID" }
+                        }
+                    },
+                    "Opacitymap": {
```

Comment:
I'm assuming this will need to be changed as opacity maps aren't VisAssets. Not sure what I should change it to though...